### PR TITLE
mention cromwell version upgrade

### PR DESCRIPTION
**Pull request by wshands**

### CODE REVIEW

**Review by coverbeck — APPROVED** [review]

File: docs/advanced-topics/dockstore-cli/advanced-features.rst [hunk 1]
```diff
@@ -330,14 +330,15 @@ with the `Cromwell <https://github.com/broadinstitute/cromwell>`__ version liste
...
+|     1.12    |          77           |
++-------------+-----------------------+
 
 Additionally, you can override the cromwell version in your
 ``~/.dockstore/config`` using for example:
 
 ::
 
-    cromwell-version = 58
+    cromwell-version = 78
```
Comment (NatalieEO) [inline docs/advanced-topics/dockstore-cli/advanced-features.rst:341]: 77?

**Review by NatalieEO — CHANGES_REQUESTED** [review]

File: docs/advanced-topics/dockstore-cli/advanced-features.rst (hunk 1 above)
Comment (wshands) [reply docs/advanced-topics/dockstore-cli/advanced-features.rst:341]: its an example of bumping the cromwell version to a newer version than what is natively supported in Dockstore. So if there was a version 78 of Cromwell, you could use that in the CLI by overriding the Cromwell version.

Comment (aofarrel) [reply docs/advanced-topics/dockstore-cli/advanced-features.rst:341]: To decrease confusion, I think we should consider putting in a previous version, even though that's generally not something someone would do.

Comment (aofarrel) [reply docs/advanced-topics/dockstore-cli/advanced-features.rst:341]: Also: Let's say someone sets cromwell-version = 60. Would the Dockstore CLI download Cromwell 60 on the fly or would the JAR need to already exist in a certain directory? Whichever of these happens should probably be briefly documented.

**Review by aofarrel — CHANGES_REQUESTED** [review]

File: docs/advanced-topics/dockstore-cli/advanced-features.rst (hunk 1 above)
Comment (wshands) [reply docs/advanced-topics/dockstore-cli/advanced-features.rst]: fixed

**Review by NatalieEO — APPROVED** [review]
sorry, I had a brain fart when reviewing it before lol

**Review by aofarrel — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]
after build success

**Comment by denis-yuen** [comment]
(oops)

**Pull request merged**